requirements/pip.txt

```diff
@@ -10,5 +10,5 @@ wheel==0.37.1
 # The following packages are considered to be unsafe in a requirements file:
 pip==22.0.3
     # via -r requirements/pip.in
-setuptools==60.7.1
+setuptools==60.8.1
```

Comment:
This is using setuptools 60 but we have pinned it to <60 in common constraints. We should add the constraint to pip.in file here

Reply:
ok